src/electron/biometric.windows.main.ts

```diff
@@ -110,4 +110,16 @@ export default class BiometricWindowsMain implements BiometricMain {
         } catch { /*Ignore error*/ }
         return [];
     }
+
+    getWindowsMajorVersion(): number {
+        if (process.platform !== 'win32') {
+            return null;
```

Comment:
it seems weird to me to return null on something that expects a number. maybe that's coming from my c# senses. could this return -1 or something instead?

Reply:
Yes, I can do that.